MAINTAINERS

```diff
@@ -0,0 +1,78 @@
+
+
+	List of maintainers and how to submit Bitcoin Core changes
+
+Please try to follow the guidelines below.  This will make things
+easier on the maintainers.  Not all of these guidelines matter for every
+trivial patch so apply some common sense.
+
+1.	Always _test_ your changes, however small.
+
+2.	Try to release a few ALPHA test versions to the net. Announce
+	them onto the bitcoin-dev channel and await results.
+
+3.	Make sure your changes compile correctly in multiple
```

Comment:
Maybe add something that it's possible and trivial to enable Travis on the users bitcoin-repository fork, so one can test the changes over Travis even before open a PR on bitcoin/bitcoin?